aten/src/ATen/native/mps/operations/MaxUnpooling.mm

```diff
@@ -0,0 +1,141 @@
+#define TORCH_ASSERT_ONLY_METHOD_OPERATORS
+#include <ATen/native/mps/MPSGraphVenturaOps.h>
+#include <ATen/native/mps/OperationUtils.h>
+
+#include <ATen/ops/max_unpool2d.h>
+#include <ATen/ops/max_unpool2d_native.h>
+
+namespace at::native {
+namespace mps {
```

Comment:
Q: why not use anonymous namespace here and get rid of all the `static` declarations?

Reply:
You are right, it is a better practice to use anonymous namespace